etc/firebase-admin.auth.api.md

```diff
@@ -264,6 +264,34 @@ export interface ProviderIdentifier {
     providerUid: string;
 }
 
+// @public
+export type RecaptchaAction = 'BLOCK';
+
+// @public (undocumented)
+export interface RecaptchaConfig {

```

Comment:
Please document this where you `export interface RecaptchaConfig`. API Extractor will pick it up from there. Thanks!